### ISSUE TITLE
Client configuration

### ISSUE THREAD

**Issue by netherfoam** [netherfoam/Titan#45]
As a developer, I'd like to configure some basic information about the client using a configuration file:

* Whether RSA is enabled
* What the public RSA and modulus is, if enabled
* What the default server's IP address is

This might be suited to loading a configuration file stored on the classpath, such as `classpath:settings.properties` or `classpath:settings.yml`, whose content might be:
```
server.rsa.enabled = true
server.rsa.key = 0x<some big number>
server.rsa.modulus = 10001
server.ip = titan.maxgamer.org
```

It'd also be nice if I could override these with command line arguments. Eg,
`java -jar Client.jar --server.ip=localhost --server.rsa.enabled=false`

**Comment by Cuzzbuzz** [comment]
Is the server perhaps up and running?

**Comment by netherfoam** [comment]
It is now

**Comment by Cuzzbuzz** [comment]
Dont have the client anymore, else I would have checked it out for sure!

**Comment by netherfoam** [comment]
It's still in bitbucket, plus the client is still compiled at
jenkins.maxgamer.org and downloadable lol

**Comment by Cuzzbuzz** [comment]
Yeah, I got the client again from the bitbucket repo. (I think the .jar at jenkins has the local ip as default) 

But still at the same problem of not being able to login. What does the console print?

**Comment by netherfoam** [comment]
You can use the "servaddr titan.maxgamer.org" args for the process and
it'll point you at the server no matter the client

Can't check what the server prints though, I'm not at home

**Comment by Cuzzbuzz** [comment]
--> "Error connecting to server" 
Dont worry about it! Only wanted to check out your recent progress! 

Forgot to ask - not completely, because I sent some messages in the Skype group, but no one ever replied :p - How is life since it has been awhile now

**Comment by netherfoam** [comment]
Haha yeah, been good. Still programming at work. How about yourself?

I haven't used Skype in ages anyway

**Comment by Cuzzbuzz** [comment]
Good to hear! I'm doing my 4th year at uni now. Currently writing my final thesis and have some internships at tax consultancy firms planned for summer. Haven't seen code in ages, but those firms plan on launching a "tax digitalising taskforce" where software engineers and tax specialists will work together. So yeah, I really look forward to summer!

**Comment by netherfoam** [comment]
Huh, sounds like a neat opportunity! Best of luck :)

**Comment by Cuzzbuzz** [comment]
Online now. Looks great so far!

**Comment by netherfoam** [comment]
Fingers crossed it stays fine now. Have had connection issues with the
database forever

**Comment by Cuzzbuzz** [comment]
Client code is broken. Only safe mode available. Was like: why only 10 fps :P

**Comment by netherfoam** [comment]
Really? Try go back a few versions and see if the issue persists. I always
had shocking performance from the client, and it's improved recently,
probably a change in hardware

**Comment by Cuzzbuzz** [comment]
I'm quite sure the last version I committed, worked entirely. Not sure what you have changed/reverted. I might take a look this weekend!
